### PR TITLE
Small fixes

### LINKED ISSUE #293: Redesign: Thank you for contacting us message
At the moment in the redesign when a message is sent via the contact form, we get this:
screen shot 2017-06-23 at 20 39 21

The styling here probably needs some more thought! Even if it's just moving the second sentence onto a separate line.

### PR DESCRIPTION
**Pull request by Cleop**
#293 Styling tweaks to the contact us section success message
#291 Adds urls to EMF case study page
#281 Aligns Portfolio title with boxes

### CODE REVIEW

File: redesign/cs-emf.html [hunk 1]
```diff
@@ -55,11 +55,11 @@ <h1 class="dwyl-dark-gray f3-plus w-90 tc center">Online Membership Platform</h1
       <article class="w-50-ns dtc v-top istok-web lh-copy dwyl-dark-gray">
         <h3 class="b mb2">Project: <span class="normal">CE100</span></h3>
         <div class="mb4">
-          <a href="#" class="dib link mv2 pa2 dwyl-bg-yellow dwyl-dark-gray br2 f5 shadow-4">
+          <a href="https://www.ellenmacarthurfoundation.org/ce100" class="dib link mv2 pa2 dwyl-bg-yellow dwyl-dark-gray br2 f5 shadow-4">
```
Comment (iteles) [inline redesign/cs-emf.html]: This is not the URL to the live app, but we also don't have permissions to share this yet, so I would comment this `<a>` out altogether for now. GH is open source so 👍

File: redesign/blog.html [hunk 2]
```diff
@@ -142,9 +142,9 @@ <h2 class="tr white"><span class="dwyl-dark-gray lh-copy fw9">Address:</span> 16
             <i class="fa fa-paper-plane"></i>&nbsp; Say Hello</button>
         </form>
 
-        <div style="display:none;" id="thankyou_message">
-          <h2>
-            <em>Thanks</em> for contacting us! We will get back to you soon!</h2>
+        <div style="display:none;" class="tc pb4" id="thankyou_message">
+          <h2 class="">Thanks for contacting us!</h2>
```
Comment (iteles) [inline redesign/blog.html]: Is there a need for `class=""` if we're not actually using a class?

File: redesign/cs-emf.html (hunk 1 above)
Comment (Cleop) [reply redesign/cs-emf.html]: Apologies, this is what @markwilliamfirth asked me to put in: https://github.com/dwyl/dwyl-site/issues/291#issuecomment-324310716

Comment (markwilliamfirth) [reply redesign/cs-emf.html]: @iteles can you add the correct link to the case study content doc?
https://github.com/dwyl/hq/issues/377

**Pull request merged**